python/taichi/lang/ast/ast_transformer.py

```diff
@@ -717,7 +721,15 @@ def build_Attribute(ctx, node):
             node.ptr = lambda val: append(x.parent(), index, val)
         else:
             build_stmt(ctx, node.value)
-            node.ptr = getattr(node.value.ptr, node.attr)
+            if isinstance(node.value.ptr,
+                          Expr) and not hasattr(node.value.ptr, node.attr):
+                # pylint: disable-msg=C0415
+                from taichi.lang import matrix_ops as tensor_ops
```

Comment:
let's put all the `import` on the top. Otherwise it's gonna introduce extra efforts to debug module name conflicts or circular imports.

Reply:
Currently there will be a circular import error
(`ast_transformer` <- `kernel_impl` <- `matrix_ops` <- `ast_transformer`) if we put it on the top.